.github/workflows/docs_stable.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Docs for releases
+
+on:
+  workflow_dispatch: # run on request (no need for PR)
+  release:
+    types: [published]
+
+jobs:
+  Build-Docs:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: python -m pip install tox
+      - name: Build-Docs
+        run: |
+          echo RELEASE_VERSION=${GITHUB_REF#refs/*/} >> $GITHUB_ENV
+          tox -e build-doc
+      # - name: Deploy
+      #   uses: peaceiris/actions-gh-pages@v3
+      #   with:
+      #     github_token: ${{ secrets.GITHUB_TOKEN }}
+      #     publish_dir: ./public
+      #     destination_dir: ${{ env.RELEASE_VERSION }}
+      #     force_orphan: true
```

Comment:
Do you have a plan to use this in the near future?

Reply:
yes. once it merged, I can trigger this wf to check whether current docs build env can work on the GH actions-runner instance. after checking it, I will enable this job.